storage/libradb/src/lib.rs

```diff
@@ -483,37 +483,31 @@ impl DbReader for LibraDB {
         start_epoch: u64,
         end_epoch: u64,
     ) -> Result<EpochChangeProof> {
-        let _timer = LIBRA_STORAGE_API_LATENCY_SECONDS
-            .with_label_values(&["get_epoch_ending_ledger_infos"])
-            .start_timer();
-
-        let (ledger_info_with_sigs, more) =
-            Self::get_epoch_ending_ledger_infos(&self, start_epoch, end_epoch)?;
-        Ok(EpochChangeProof::new(ledger_info_with_sigs, more))
+        gauged_api("get_epoch_ending_ledger_infos", || {
```

Comment:
waaah, what's the benefit of this approach versus say...

```
let _timer = start_storage_timer(name);

run_op
```

maybe one additional lines of code but the following is a little interesting... though I think safety rules also looks a little interesting now: https://github.com/libra/libra/blob/master/consensus/safety-rules/src/safety_rules.rs#L442 <3

code is fine, but figure let's chat

Reply:
It times "Ok"s and "Err"s seperately.

Reply:
ahhh, fancy